lib/pdk/cli/util.rb

```diff
@@ -48,9 +48,30 @@ def prompt_for_yes(question_text, opts = {})
       end
       module_function :prompt_for_yes
 
+      # Uses environment variables to detect if the current process is running in common
+      # Continuous Integration (CI) environments
+      # @return [Boolean] Whether the PDK is in a CI based environment
+      def ci_environment?
+        [
+          'APPVEYOR_BUILD_FOLDER', # AppVeyor CI
+          'GITLAB_CI',             # GitLab CI
+          'JENKINS_URL',           # Jenkins
+          'BUILD_DEFINITIONNAME',  # Azure Pipelines
+          'TEAMCITY_VERSION',      # Team City
+          'BAMBOO_BUILDKEY',       # Bamboo
+          'GOCD_SERVER_URL',       # Go CD
+          'TRAVIS',                # Travis CI
+          'GITHUB_WORKFLOW'        # GitHub Actions
```

Comment:
Travis-CI also set some ["generic" environment variables](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables):

* `CI=true`
* `CONTINUOUS_INTEGRATION=true`

Does it make sense to add these to the list hopping that other CI systems use such variables?

Reply:
Yeah wouldn't hurt, I'll amend those in.